### PR TITLE
Update MetaDataValidator.scala 

### PR DESCRIPTION
**Pull request by n-niibe**
The fourth argument to the constructor of CSVReader is "escape character".

In this modification, you will be double quotes of escape can be correctly.

reference:
http://opencsv.sourceforge.net/apidocs/com/opencsv/CSVReader.html#CSVReader(java.io.Reader, char, char, char)

### CODE REVIEW

**Comment by DavidUnderdown** [comment]
Apologies for the delay in acknowledging this pull request.  I've now assigned it to a couple of our devs for review.

File: csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/MetaDataValidator.scala [hunk 1]
```diff
@@ -90,10 +90,9 @@ trait MetaDataValidator {
         CSVParser.DEFAULT_QUOTE_CHARACTER
     }
 
-    //TODO CSVReader does not appear to be RFC 4180 compliant as it does not support escaping a double-quote with a double-quote between double-quotes
```
Comment (adamretter) [inline csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/MetaDataValidator.scala]: Why did you remove this TODO?

File: csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/MetaDataValidator.scala [hunk 2]
```diff
@@ -90,10 +90,9 @@ trait MetaDataValidator {
...
     //TODO CSVReader does not seem to allow you to enable/disable quoted columns
     //we need a better CSV Reader!
-    (managed(new CSVReader(csv, separator, CSVParser.DEFAULT_QUOTE_CHARACTER, CSVParser.NULL_CHARACTER)) map {
+    (managed(new CSVReader(csv, separator, CSVParser.DEFAULT_QUOTE_CHARACTER, CSVParser.DEFAULT_ESCAPE_CHARACTER)) map {
       reader =>
```
Comment (adamretter) [inline csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/MetaDataValidator.scala]: This looks reasonable to me.

File: csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/MetaDataValidator.scala (hunk 1 above)
Comment (n-niibe) [inline csv-validator-core/src/main/scala/uk/gov/nationalarchives/csv/validator/MetaDataValidator.scala]: Thank you for your review!

The reason that I have to delete the comment, 
In my understanding, 
If the escaped double quotation marks such as [\"], CSVReader supports double quotation marks. (Isn't this based upon RFC 4180?)

For example the following CSV is possible to verify.

<*>CSV Schema
version 1.0
@totalColumns  2 @separator ',' @quoted
Name:
Age:

<*>CSV data
"This is \"OK\"","10"


<*>Remarks
Backslash must be escaped.
[\] -> [\\]

**Comment by n-niibe** [comment]
I canceled this pull request.
